apps/frontend/src/modules/groups/slices/actions.ts

```diff
@@ -15,4 +16,16 @@ const loadAll = createAsyncThunk<
 	return await groupApi.getAll();
 });
 
-export { loadAll };
+const deleteById = createAsyncThunk<number, number, AsyncThunkConfig>(
```

Comment:
Maybe just `delete`? We  don't have another version of deleting, just by Id

Reply:
If shortly, it's a reserved keyword